web/src/providers/save-workflow.tsx

```diff
@@ -42,7 +42,7 @@ export function SaveWorkflowProvider({
 
 	const handleSaveWorkflow = async () => {
 		if (isPending) {
-			return;
+			return false;
```

Comment:
I think it is okay to remove the if-condition. It is probably more dangerous to return `false` in this case because if we would do error handling based on the result, it would trigger an error although there is none.